extension/script.js

```diff
@@ -394,10 +380,12 @@ function getNearestLink(element) {
 
 function processTitle(titleContainer) {
   if (titleContainer.hasListener) return;
-  Object.assign(titleContainer.style, {
-    display: 'flex',
-    gap: '15px',
-  });
+  titleContainer.classList.add("ta-title-container");
+
+  if (titleContainer.style.display === "-webkit-box") {
+    // Compatibility with DeArrow
+    titleContainer.style.setProperty("display", "flex", "important");
+  }
```

Comment:
Depending on which one gets to modifying it first, one of them will be done first. I have put this compatibility one on both sides. Only visible titles will get `-webkit-box`, once a title needs to be made invisible, it will correctly override TubeArchivist's modification